torq.sh

```diff
@@ -139,7 +144,9 @@ checkinput() {
   PROCS=$(awk -F, '{if(NR>1) print $4}' "$CSVPATH")                                                 # get all process names from csv
   avail=()
   for i in $input; do
-    if [[ $(echo "$PROCS" | grep -w "$i") ]]; then                                                  # check input process is valid
+    if [[ $i == "-force" ]]; then                                                                   # check for force flag
+      echo "Force stop has been set"
```

Comment:
As per offline convo:
```suggestion
      :
```